odk1-src/central-forms.rst

```diff
@@ -122,6 +161,40 @@ To set the form lifecycle stage, go to the :ref:`Form Access <central-projects-f
 
 You can find more information about the Form Access page :ref:`here <central-projects-form-access>`.
 
+.. _central-forms-updates:
+
```

Comment:
I think it would be helpful to have an additional section about submissions. What submissions are expected in the submission listing, export, OData? What if fields are deleted/added? How does the version of the corresponding form version appear in those places?

Reply:
yeah, this information is definitely missing right now.. but does it go here or on the submissions page? also i don't understand your last question.

Reply:
I think putting it here makes sense because it means someone who has been using Central already can read this section and know everything about how form updates affect them.

Yes, that last question is poorly-worded. I thought that the form version for each submission would be included in the submission table and in the exports but it's not. We should briefly discuss whether it should be added in a future version but no action needed for docs.

Reply:
yeah, it's not included because Briefcase doesn't do it and i've tried not to deviate from that format too much